database_schemas/cassandra/benchflow.cql

```diff
@@ -210,128 +209,175 @@ CREATE TABLE exp_process_duration (
 );
 
 CREATE TABLE exp_cpu (
-  cpu_weighted_avg float,
-  cpu_q1_min float,
-  cpu_q1_max float,
-  cpu_q2_min float,
-  cpu_q2_max float,
-  cpu_q3_min float,
-  cpu_q3_max float,
-  cpu_p95_min float,
-  cpu_p95_max float,
-  cpu_ci095_min float,
-  cpu_ci095_max float,
-  cpu_median_min float,
-  cpu_median_max float,
-  cpu_mode_min float,
-  cpu_mode_max float,
-  cpu_min float,
-  cpu_max float,
-  cpu_sd float,
+  cpu_weighted_avg double,
+  cpu_q1_min double,
+  cpu_q1_max double,
+  cpu_q2_min double,
+  cpu_q2_max double,
+  cpu_q3_min double,
+  cpu_q3_max double,
+  cpu_p95_min double,
+  cpu_p95_max double,
+  cpu_ci095_min double,
+  cpu_ci095_max double,
+  cpu_median_min double,
+  cpu_median_max double,
+  cpu_mode_min double,
+  cpu_mode_max double,
+  cpu_min double,
+  cpu_max double,
+  cpu_sd double,
   cpu_best list<text>,
   cpu_average list<text>,
   cpu_worst list<text>,
   experiment_id text,
+  container_id text,
   PRIMARY KEY ((experiment_id))
 );
 
 CREATE TABLE exp_ram (
-  ram_weighted_avg float,
-  ram_q1_min float,
-  ram_q1_max float,
-  ram_q2_min float,
-  ram_q2_max float,
-  ram_q3_min float,
-  ram_q3_max float,
-  ram_p95_min float,
-  ram_p95_max float,
-  ram_ci095_min float,
-  ram_ci095_max float,
-  ram_median_min float,
-  ram_median_max float,
-  ram_mode_min float,
-  ram_mode_max float,
-  ram_min float,
-  ram_max float,
-  ram_sd float,
+  ram_weighted_avg double,
+  ram_q1_min bigint,
```

Comment:
You never saw it because Docker stats provide you these data using the amount of bytes, but then you perform a conversion to MB. And by the way, I suspect the following two conversion should be performed with a `double` precision: https://github.com/benchflow/data-transformers/blob/dev/data-transformers/transformers/statsTransformer.py#L71